[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron -knative-prow-robot

/cc knative-sandbox/client-writers
/assign knative-sandbox/client-writers

Produced by: knative-sandbox/knobots/actions/update-deps

Details:
```
bumping knative.dev/client d0327bc...67bbe37:
  > 67bbe37 Update community files (# 1332)
  > 3a97aa2 upgrade to latest dependencies (# 1330)
  > aacd22c Update actions (# 1331)
  > 350b2de upgrade to latest dependencies (# 1324)
  > 0c6ef82 Fix Subscription's Channel to use KRefence type (# 1326)
bumping knative.dev/hack 93ad912...de04b70:
  > de04b70 Do not print debug info when running release.sh (# 70)
bumping knative.dev/serving 5a0ea08...966f015:
  > 966f015 Update net-kourier nightly (# 11454)
  > e297718 drop use of KubeClient (# 11432)
  > 16a1f6f Update net-certmanager nightly (# 11453)
  > f61441d Update net-istio nightly (# 11452)
  > cb81f43 Update net-contour nightly (# 11451)
  > aa2d324 Make schema generation idempotent (# 11445)
  > 561868d Use TLS for websocket and gRPC tests (# 11322)
  > a5caa66 upgrade to latest dependencies (# 11449)
  > 3676663 Regenerate schema after changing some comments in pkg (# 11447)
  > f0520a8 Update community files (# 11446)
  > 2a1b1e7 upgrade to latest dependencies (# 11444)
  > c6d95c0 Unflake per-item test again (# 11443)
  > 22f9ebf Update net-istio nightly (# 11416)
  > 4530c92 Wrap the Reconciler context with timeout (# 10587)
  > c1051d9 Update net-kourier nightly (# 11442)
  > 07e6fc1 Move schemapatch config here for easier access (# 11407)
  > fd235d8 Update net-contour nightly (# 11441)
  > d920d2a drop dependency on knative.dev/networking/test (# 11439)
  > 40a36e9 Update net-certmanager nightly (# 11435)
  > b417fc3 upgrade to latest dependencies (# 11437)
  > 248381d Update community files (# 11438)
  > 65929cc Update net-kourier nightly (# 11436)
  > d09e5e3 Update net-contour nightly (# 11434)
  > 526721f Improve readiness tests for exec case (# 11425)
  > a9aeafb Fix PerItem test flake (# 11430)
  > 64a0bb5 upgrade to latest dependencies (# 11433)
  > 620cb44 Update net-contour nightly (# 11428)
  > f89b0de Update net-certmanager nightly (# 11427)
  > b6fa655 Update net-kourier nightly (# 11426)
  > 27f036c Update net-certmanager nightly (# 11414)
  > f98b448 Use cert-manager API version to v1 for test resources (# 11424)
  > 27e330a Update net-contour nightly (# 11415)
  > df3b056 fix typo and redundant code (# 11419)
  > ecca088 Update actions (# 11421)
  > 4265285 upgrade to latest dependencies (# 11420)
  > 8a9bb8e upgrade to latest dependencies (# 11418)
  > 5d1672d Update net-kourier nightly (# 11417)
  > 9bb6349 Test that readiness is not reported early in HTTP probe test (# 11402)
  > 61178b2 upgrade to latest dependencies (# 11412)
  > 8eb5b88 Update net-istio nightly (# 11397)
  > 01f1cb6 Remove myself following-up knative/community# 628 (# 11408)
  > 1cd1e71 Have our e2e tests use K8s 1.19 again (# 11384)
  > 71e501e upgrade to latest dependencies (# 11405)
  > f9356e3 Update schema generator as a bug has been fixed (# 11406)
  > fc8d0c8 Allow the autoscale upgrade test traffic to settle before upgrading (# 11403)
  > 77d32d8 Regenerate schemas to pick up capabilities addition (# 11400)
  > efbc518 Update net-contour nightly (# 11396)
  > 15e067e Update net-kourier nightly (# 11401)
  > 3afc6e4 Move activator count computation into KPA (# 11375)
  > 8cf7372 Add readiness test image (# 11398)
  > 0760219 Set spec.HTTPOption in Ingress created from DomainMapping (# 11399)
  > 9cd446e Update net-kourier nightly (# 11392)
  > fcaccd6 adding securityContext.capabilities.drop to schemas (# 11394)
```

maximilien reviewed (approved):
/ok-to-test
/approve
/lgtm

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/client d0327bc...67bbe37:
  > 67bbe37 Update community files (# 1332)
  > 3a97aa2 upgrade to latest dependencies (# 1330)
  > aacd22c Update actions (# 1331)
  > 350b2de upgrade to latest dependencies (# 1324)
  > 0c6ef82 Fix Subscription's Channel to use KRefence type (# 1326)
bumping knative.dev/hack 93ad912...de04b70:
  > de04b70 Do not print debug info when running release.sh (# 70)
bumping knative.dev/serving 5a0ea08...966f015:
  > 966f015 Update net-kourier nightly (# 11454)
  > e297718 drop use of KubeClient (# 11432)
  > 16a1f6f Update net-certmanager nightly (# 11453)
  > f61441d Update net-istio nightly (# 11452)
  > cb81f43 Update net-contour nightly (# 11451)
  > aa2d324 Make schema generation idempotent (# 11445)
  > 561868d Use TLS for websocket and gRPC tests (# 11322)
  > a5caa66 upgrade to latest dependencies (# 11449)
  > 3676663 Regenerate schema after changing some comments in pkg (# 11447)
  > f0520a8 Update community files (# 11446)
  > 2a1b1e7 upgrade to latest dependencies (# 11444)
  > c6d95c0 Unflake per-item test again (# 11443)
  > 22f9ebf Update net-istio nightly (# 11416)
  > 4530c92 Wrap the Reconciler context with timeout (# 10587)
  > c1051d9 Update net-kourier nightly (# 11442)
  > 07e6fc1 Move schemapatch config here for easier access (# 11407)
  > fd235d8 Update net-contour nightly (# 11441)
  > d920d2a drop dependency on knative.dev/networking/test (# 11439)
  > 40a36e9 Update net-certmanager nightly (# 11435)
  > b417fc3 upgrade to latest dependencies (# 11437)
  > 248381d Update community files (# 11438)
  > 65929cc Update net-kourier nightly (# 11436)
  > d09e5e3 Update net-contour nightly (# 11434)
  > 526721f Improve readiness tests for exec case (# 11425)
  > a9aeafb Fix PerItem test flake (# 11430)
  > 64a0bb5 upgrade to latest dependencies (# 11433)
  > 620cb44 Update net-contour nightly (# 11428)
  > f89b0de Update net-certmanager nightly (# 11427)
  > b6fa655 Update net-kourier nightly (# 11426)
  > 27f036c Update net-certmanager nightly (# 11414)
  > f98b448 Use cert-manager API version to v1 for test resources (# 11424)
  > 27e330a Update net-contour nightly (# 11415)
  > df3b056 fix typo and redundant code (# 11419)
  > ecca088 Update actions (# 11421)
  > 4265285 upgrade to latest dependencies (# 11420)
  > 8a9bb8e upgrade to latest dependencies (# 11418)
  > 5d1672d Update net-kourier nightly (# 11417)
  > 9bb6349 Test that readiness is not reported early in HTTP probe test (# 11402)
  > 61178b2 upgrade to latest dependencies (# 11412)
  > 8eb5b88 Update net-istio nightly (# 11397)
  > 01f1cb6 Remove myself following-up knative/community# 628 (# 11408)
  > 1cd1e71 Have our e2e tests use K8s 1.19 again (# 11384)
  > 71e501e upgrade to latest dependencies (# 11405)
  > f9356e3 Update schema generator as a bug has been fixed (# 11406)
  > fc8d0c8 Allow the autoscale upgrade test traffic to settle before upgrading (# 11403)
  > 77d32d8 Regenerate schemas to pick up capabilities addition (# 11400)
  > efbc518 Update net-contour nightly (# 11396)
  > 15e067e Update net-kourier nightly (# 11401)
  > 3afc6e4 Move activator count computation into KPA (# 11375)
  > 8cf7372 Add readiness test image (# 11398)
  > 0760219 Set spec.HTTPOption in Ingress created from DomainMapping (# 11399)
  > 9cd446e Update net-kourier nightly (# 11392)
  > fcaccd6 adding securityContext.capabilities.drop to schemas (# 11394)

Signed-off-by: Knative Automation <automation@knative.team>

maximilien reviewed (approved):
/ok-to-test
/approve
/lgtm
